### PR TITLE
SDL MT Cloud: Provide user with ability to reevaluate translations

### PR DESCRIPTION
**Pull request by ealbu**

### CODE REVIEW

File: SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Service/TranslationService.cs [hunk 1]
```diff
@@ -97,9 +99,9 @@ public async Task<SubscriptionInfo> GetLanguagePairs()
 			return await _httpClient.GetResult<SubscriptionInfo>(response);
 		}
 
-		public async Task<HttpResponseMessage> SendFeedback(SegmentId? segmentId, dynamic rating, string originalText, string improvement)
+		public async Task<HttpResponseMessage> SendFeedback(SegmentId? segmentId, dynamic rating, string originalText, string improvement, QualityEstimation qualityEstimation)
```
Comment (andreaGhisa) [inline SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Service/TranslationService.cs:101]: Should we consider to use a object for all this properties something like "Feedbackmodel"? this method has to many parameters.

File: SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Service/TranslationService.cs [hunk 2]
```diff
@@ -235,7 +237,7 @@ private async Task<(HttpResponseMessage, string)> CheckTranslationStatus(string
 			return (await GetTranslationResult(id), qualityEstimation);
 		}
 
-		private dynamic CreateFeedbackRequest(SegmentId? segmentId, dynamic rating, string originalText, string improvement)
+		private dynamic CreateFeedbackRequest(SegmentId? segmentId, dynamic rating, string originalText, string improvement, QualityEstimation qualityEstimation)
```
Comment (andreaGhisa) [inline SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Service/TranslationService.cs:239]: same here.

**Review by andreaGhisa — CHANGES_REQUESTED** [review]

File: SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Service/TranslationService.cs (hunk 1 above)
Comment (ealbu) [reply SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Service/TranslationService.cs]: done

File: SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Service/TranslationService.cs (hunk 2 above)
Comment (ealbu) [reply SDLMTCloud.Provider/Sdl.Community.MTCloud.Provider/Service/TranslationService.cs]: done

**Review by andreaGhisa — APPROVED** [review]

**Review by andreaGhisa — APPROVED** [review]

**Pull request merged**